frameworks/hdfs/docs/security.md

```diff
@@ -0,0 +1,145 @@
+```
+<service primary>/name-0-node.<service subdomain>.autoip.dcos.thisdcos.directory@<service realm>
+<service primary>/name-0-zkfc.<service subdomain>.autoip.dcos.thisdcos.directory@<service realm>
+<service primary>/name-1-node.<service subdomain>.autoip.dcos.thisdcos.directory@<service realm>
+<service primary>/name-1-zkfc.<service subdomain>.autoip.dcos.thisdcos.directory@<service realm>
+
+<service primary>/journal-0-node.<service subdomain>.autoip.dcos.thisdcos.directory@<service realm>
+<service primary>/journal-1-node.<service subdomain>.autoip.dcos.thisdcos.directory@<service realm>
+<service primary>/journal-2-node.<service subdomain>.autoip.dcos.thisdcos.directory@<service realm>
+
+<service primary>/data-<data-index>-node.<service subdomain>.autoip.dcos.thisdcos.directory@<service realm>
+
+```
+with:
+- `service primary = service.security.kerberos.primary`
+- `data index = 0 up to data_node.brokers.count - 1`
+- `service subdomain = service.name with all `/`'s removed`
+- `service realm = service.security.kerberos.realm`
+
+For example, if installing with these options:
+```json
+{
+    "service": {
+        "name": "a/good/example",
+        "kerberos": {
+            "primary": "example",
+            "realm": "EXAMPLE"
+        }
+    },
+    "data_node": {
+        "count": 3
+    }
+}
+```
+then the principals to create would be:
+```
+example/name-0-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/name-0-zkfc.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/name-1-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/name-1-zkfc.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+
+example/journal-0-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/journal-1-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/journal-2-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+
+example/data-0-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/data-1-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/data-2-node.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+```
+
+{% include services/security-configure-transport-encryption.md
+    techName="Apache HDFS" %}
+
+#### Install the Service
+
+Install the DC/OS Apache HDFS service with the following options in addition to your own:
+```json
+{
+    "service": {
+        "kerberos": {
+            "enabled": true,
+            "kdc": {
+                "hostname": "<kdc host>",
+                "port": <kdc port>
+            },
+            "primary": "<service primary default hdfs>",
+            "realm": "<realm>",
+            "keytab_secret": "<path to keytab secret>",
+            "debug": <true|false default false>
+        }
+    }
+}
+```
+
+<!-- TO BE DETERMINED *Note*: It is possible to enable Kerberos after initial installation but the service may be unavailable during the transition. Additionally, your HDFS clients will need to be reconfigured. -->
```

Comment:
Is there a JIRA tracking this TBD?

Reply:
Yes, there is a JIRA. https://jira.mesosphere.com/browse/INFINITY-3036